integration-tests/src/main/resources/application.properties

```diff
@@ -0,0 +1,9 @@
+quarkus.application.name=JasperReports IT
+quarkus.native.resources.includes=*.xml,*.jrxml
```

Comment:
I feel like in the Processor we should always add all *.xml, *.jrxml and *.jasper files to the native image?

Reply:
Excellent @nderwin yes those too